### PR TITLE
Enable flake8 on Travis CI

### PR DESCRIPTION
**Pull request by winniehell**
## Description of the issue/feature this PR addresses

Enables automated flake8 check in Travis CI and fixes existing violations.

see also https://github.com/senaite/senaite.core/pull/2077

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,5 +16,8 @@ before_install:
   - bin/buildout -N -t 3 annotate
 install:
   - bin/buildout -N -t 3
+before_script:
+  - bin/pip install flake8
 script:
+  - bin/flake8 --config travis_ci_flake8.cfg src/
```
Comment (winniehell) [inline .travis.yml:22]: it looks like Travis CI currently doesn't run on pull requests, so this change is not tested

File: travis_ci_flake8.cfg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+# temporarily disable violations for existing code until they are cleaned up
+extend-ignore =
+    # E501: line too long
```
Comment (winniehell) [inline travis_ci_flake8.cfg:4]: This has roughly 60 violations, so I didn't want to clean them up in this pull request. Does this project generally stick to the 79 characters limit?

**Comment by winniehell** [comment]
@ramonski you may have missed this pull request. can I ask you to take a look?

File: travis_ci_flake8.cfg (hunk 2 above)
Comment (ramonski) [reply travis_ci_flake8.cfg:4]: Hi @winniehell, thanks for all your efforts in cleaning up the codebases.
Yes, it *should* actually stick to the 79 characters limit, but I'm not sure with the route decorators, which might exceed this limit here and there...

File: .travis.yml (hunk 1 above)
Comment (ramonski) [reply .travis.yml:22]: At least we can run it manually against the configuration, so no worries.

**Review by ramonski — APPROVED** [review]

**Pull request merged**